flow360/component/simulation/simulation_params.py

```diff
@@ -244,6 +245,12 @@ def check_consistency_wall_function_and_surface_output(cls, v):
         """Only allow wallFunctionMetric output field when there is a Wall model with a wall function enabled"""
         return _check_consistency_wall_function_and_surface_output(v)
 
+    @pd.model_validator(mode="after")
+    @classmethod
+    def check_cht_solver_settings(cls, v):
+        """Check the Conjugate Heat Transfer settings, transferred from checkCHTSolverSettings"""
+        return _check_cht_solver_settings(v)
```

Comment:
don't use single letter variables names
```suggestion
    def check_cht_solver_settings(cls, value):
        """Check the Conjugate Heat Transfer settings, transferred from checkCHTSolverSettings"""
        return _check_cht_solver_settings(value)
```

can you run pylint on this file? It should be caught by pylint

Reply:
I ran pylint on this file before I submitted it. There are other functions that has single variable names but are not caught by pylint.